### PR TITLE
Fix part #1071: High-fi of Topic in progress Landscape mode

### PR DESCRIPTION
**Pull request by veena14cs**
## Explanation
This PR focuses on hi-fi part of #1071 topic in progress Landscape mode. This Pr also fixes testcases.

### Mock
https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/ce21cbf2-0439-47d7-bb37-005cfc6b9742/UP-Topics-in-Progress-/

- [x] Passes Accessibility test

### Screenshot
Screenshot_20200521-123134

### CODE REVIEW

File: app/src/main/res/layout-land/ongoing_topic_list_fragment.xml [hunk 1]
```diff
@@ -52,6 +52,8 @@
         android:layout_height="match_parent"
         android:clipToPadding="false"
         android:overScrollMode="never"
+        android:paddingStart="56dp"
+        android:paddingEnd="56dp"
         android:paddingTop="28dp"
         android:paddingBottom="108dp"
```
Comment (rt4914) [inline app/src/main/res/layout-land/ongoing_topic_list_fragment.xml]: This should be `88dp`

File: app/src/main/res/layout-land/ongoing_topic_item.xml [hunk 2]
```diff
@@ -14,68 +14,79 @@
...
-    android:layout_marginStart="96dp"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="16dp"
     android:layout_marginTop="8dp"
-    android:layout_marginBottom="8dp"
-    android:layout_marginEnd="96dp"
-    app:cardBackgroundColor="@color/white"
-    app:cardCornerRadius="4dp"
-    app:cardElevation="4dp">
+    android:layout_marginEnd="16dp"
```
Comment (rt4914) [inline app/src/main/res/layout-land/ongoing_topic_item.xml:22]: Add elevation `4dp`

File: app/src/main/res/layout-land/ongoing_topic_item.xml [hunk 3]
```diff
@@ -14,68 +14,79 @@
...
-        android:layout_width="80dp"
+        android:layout_width="0dp"
         android:layout_height="0dp"
-        android:layout_marginStart="8dp"
+        android:background="@color/topicThumbnailBackground"
+        android:importantForAccessibility="no"
         android:scaleType="centerInside"
         android:src="@{viewModel.topic.topicThumbnail.thumbnailGraphic}"
-        app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintDimensionRatio="16:9"
```
Comment (rt4914) [inline app/src/main/res/layout-land/ongoing_topic_item.xml]: This ratio should be `4:3`

**Review by rt4914 — CHANGES_REQUESTED** [review]
PTAL

File: app/src/main/res/layout-land/ongoing_topic_item.xml (hunk 3 above)
Comment (veena14cs) [reply app/src/main/res/layout-land/ongoing_topic_item.xml]: Done.

File: app/src/main/res/layout-land/ongoing_topic_item.xml (hunk 2 above)
Comment (veena14cs) [reply app/src/main/res/layout-land/ongoing_topic_item.xml:22]: Updated.

File: app/src/main/res/layout-land/ongoing_topic_list_fragment.xml (hunk 1 above)
Comment (veena14cs) [reply app/src/main/res/layout-land/ongoing_topic_list_fragment.xml]: Done.

File: app/src/main/res/layout-land/ongoing_topic_item.xml [hunk 4]
```diff
@@ -14,68 +14,80 @@
   </data>
 
   <com.google.android.material.card.MaterialCardView
+    android:id="@+id/topic_container"
     android:layout_width="match_parent"
-    android:layout_height="92dp"
-    android:layout_marginStart="96dp"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="16dp"
     android:layout_marginTop="8dp"
```
Comment (rt4914) [inline app/src/main/res/layout-land/ongoing_topic_item.xml]: I missed this in last review. This should be `12dp` as the gap between two items is `24dp`

File: app/src/main/res/layout-land/ongoing_topic_list_fragment.xml [hunk 5]
```diff
@@ -52,8 +52,10 @@
         android:layout_height="match_parent"
         android:clipToPadding="false"
         android:overScrollMode="never"
+        android:paddingStart="56dp"
+        android:paddingEnd="56dp"
         android:paddingTop="28dp"
-        android:paddingBottom="108dp"
+        android:paddingBottom="92dp"
```
Comment (rt4914) [inline app/src/main/res/layout-land/ongoing_topic_list_fragment.xml]: This will be `88dp` only as the bottom margin of the item is `12dp` and `100-12=88`

**Review by rt4914 — CHANGES_REQUESTED** [review]
PTAL

File: app/src/main/res/layout-land/ongoing_topic_item.xml (hunk 4 above)
Comment (veena14cs) [reply app/src/main/res/layout-land/ongoing_topic_item.xml]: I thought it was 16dp. I will correct it.

File: app/src/main/res/layout-land/ongoing_topic_list_fragment.xml (hunk 5 above)
Comment (veena14cs) [reply app/src/main/res/layout-land/ongoing_topic_list_fragment.xml]: Got it.

Comment (veena14cs) [reply app/src/main/res/layout-land/ongoing_topic_list_fragment.xml:58]: Fixed in new PR #1113.

File: app/src/main/res/layout-land/ongoing_topic_item.xml (hunk 4 above)
Comment (veena14cs) [reply app/src/main/res/layout-land/ongoing_topic_item.xml]: Done.

File: app/src/main/res/layout-land/ongoing_topic_list_fragment.xml (hunk 5 above)
Comment (veena14cs) [reply app/src/main/res/layout-land/ongoing_topic_list_fragment.xml]: Done.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Review by nikitamarysolomanpvt — APPROVED** [review]
LGTM

**Pull request merged**